src/dotnet/Chat.UI.Blazor/Components/ChatView/MessageMenu/MessageHoverMenu.razor

```diff
@@ -2,19 +2,21 @@
 @inherits MenuBase
 
 @if (_chatMessageModel != null) {
-    <MessageHoverMenuContent Message="@_chatMessageModel"/>
+    <MessageHoverMenuContent Message="@_chatMessageModel" IsSystem="@_isSystem"/>
 }
 
 @code {
     private ChatMessageModel? _chatMessageModel;
+    private bool _isSystem;
 
     [Inject] private IChats Chats { get; init; } = null!;
     [Inject] private Session Session { get; init; } = null!;
 
     protected override async Task OnParametersSetAsync() {
-        if (Arguments is not [ var sEntryId ])
+        if (Arguments is not [ var sEntryId, var sIsSystem ])
             throw new ArgumentOutOfRangeException(nameof(Arguments));
 
+        bool.TryParse(sIsSystem, out _isSystem);
```

Comment:
It's more appropriate to call bool.Parse here - to make sure it fails on a wrong MenuRef.